### PR TITLE
🪟 🚦 E2E: Fix duplicated database object warning

### LINKED ISSUE #21221: E2E: Fix "duplicated database object warning" message in logs during run tests
## Tell us about the problem you're trying to solve
Fix "duplicated database object warning" message in logs during run tests

## Describe the solution you’d like
Replaces "cypress-postgres" with "pg-promise" and improve the logic

image

### PR DESCRIPTION
**Pull request by dizel852**
## What
Resolves https://github.com/airbytehq/airbyte/issues/21221

In logs we have the warning: 
`WARNING: Creating a duplicate database object for the same connection.`
That happens because of 3rd party package we use for data manipulation in Postgres DB.
Screenshot at Dec 29 20-13-02

## How
The PR replaces `"cypress-postgres"` with `"pg-promise"` (which actually was used under the hood in "cypress-postgres")
and updates the config to close the connection with DB after querying it.
image

### CODE REVIEW

File: airbyte-webapp-e2e-tests/cypress/plugins/index.ts [hunk 1]
```diff
@@ -14,13 +14,25 @@
 
 import Cypress from "cypress";
 
+const pgp = require("pg-promise")();
+const postgresConfig = require(require("path").resolve("cypress.json"));
+
+function dbConnection(query: any, userDefineConnection: undefined) {
```
Comment (edmundito) [inline airbyte-webapp-e2e-tests/cypress/plugins/index.ts:20]: Is `undefined` the correct type for this?

File: airbyte-webapp-e2e-tests/cypress/plugins/index.ts [hunk 2]
```diff
@@ -14,13 +14,25 @@
 
 import Cypress from "cypress";
 
+const pgp = require("pg-promise")();
+const postgresConfig = require(require("path").resolve("cypress.json"));
```
Comment (edmundito) [inline airbyte-webapp-e2e-tests/cypress/plugins/index.ts:18]: This is just the `cypressConfig`?

File: airbyte-webapp-e2e-tests/cypress/plugins/index.ts (hunk 1 above)
Comment (dizel852) [reply airbyte-webapp-e2e-tests/cypress/plugins/index.ts]: Hm... Missed that! Will fix that! Thanks!

File: airbyte-webapp-e2e-tests/cypress/plugins/index.ts (hunk 2 above)
Comment (dizel852) [reply airbyte-webapp-e2e-tests/cypress/plugins/index.ts]: Oh, yeah, it's not obvious - will fix the naming! 👍

**Review by m19berger — CHANGES_REQUESTED** [review]

**Review by edmundito — APPROVED** [review]

**Review by m19berger — APPROVED** [review]
LGTM!

**Pull request merged**